mailbox/lucene/src/test/java/org/apache/james/mailbox/lucene/search/LuceneMailboxMessageSearchIndexTest.java

```diff
@@ -668,4 +668,14 @@ void notOperatorShouldReverseMatching() throws Exception {
         Stream<MessageUid> result = index.search(session, mailbox, query);
         assertThat(result).containsExactly(uid3, uid4);
     }
+
+    @Test
+    void retrieveIndexedFlagsShouldRetrieveFlags() {
```

Comment:
IMO we should have the same "contract" for `retrieveIndexedFlags` between ElasticSearch & Lucene.